src/tracking-buffer/writable-tracking-buffer.js

```diff
@@ -239,11 +251,11 @@ module.exports = class WritableTrackingBuffer {
     if (length > 0) {
       // One chunk.
       this.writeUInt32LE(length);
-      if (Buffer.isBuffer(value)) {
+      if (value instanceof Buffer) {
         this.writeBuffer(value);
       } else {
         this.makeRoomFor(length);
-        this.buffer.write(value, this.position, encoding);
+        this.buffer.write(value, this.position, this.buffer.length - this.position, encoding);
```

Comment:
Flow does not have all the different possible signatures for `Buffer.prototype.write` defined, so I had to add passing a `length` value here.

Reply:
Would this work?

```javascript
this.buffer.write(value, this.position, undefined, encoding);
```

Reply:
Unfortunately not. 😞

I'll change this to use the special `FlowFixMe` comment for now to ignore this and open a PR against flow to get it fixed there. 👍